src/lua/httpc.lua

```diff
@@ -32,8 +32,18 @@
 local driver = package.loaded.http.client
```

Comment:
Regarding this example from the commit message (1st patch):

```lua
local http = require("http")

http.encoders = {
    ['application/pdf'] = function(body, _)
        return fn(body)
    end,
}
```

I would rather add an encoder in the example (I guess that detelion of the standard encoders is the bad practice):

```lua
http.encoders['application/pdf'] = <...>
```

BTW, `'application/xml'` may be a more realistic example.

Reply:
It seems, you missed everything except the last line of my comment. Reopened so.

BTW, now the comment is also applicable for the documentation request for decoders.